test/D2L.Security.OAuth2.WebApi.IntegrationTests/TestWebService/Controllers/AuthorizationAttributeTestsController.cs

```diff
@@ -7,39 +7,48 @@ namespace D2L.Security.OAuth2.TestWebService.Controllers {
 	public sealed class AuthorizationAttributeTestsController : ApiController {
 		[HttpGet]
 		[Authentication( services: true )]
-		[Route("authorization/unspecifiedscope")]
+		[Route( "authorization/unspecifiedscope" )]
 		public void UnspecifiedScope() {
 			// Will crash because we forgot a [RequireScope(...)]	
 		}
 
 		[HttpGet]
-		[RequireScope("foo", "bar", "baz")]
-		[Route("authorization/unspecifiedprincipaltype")]
+		[RequireScope( "foo", "bar", "baz" )]
+		[Route( "authorization/unspecifiedprincipaltype" )]
 		public void UnspecifiedPrincipalType() {
 			// Will crash because we forgot an [AllowFrom(...)]	
 		}
 
 		[HttpGet]
-		[RequireScope("foo","bar","baz")]
+		[RequireScope( "foo", "bar", "baz" )]
 		[Authentication( users: true )]
-		[Route("authorization/basic")]
+		[Route( "authorization/basic" )]

```

Comment:
I don't think it's a constructive use of time to worry about people trying to infer the meaning of these tests solely from the route strings. It's pretty clear that this is just a basic test.

This test wasn't part of this PR other than my editor changing the whitespace.